cli/src/merge_tools/external.rs

```diff
@@ -55,6 +55,9 @@ impl Default for ExternalMergeTool {
     fn default() -> Self {
         Self {
             program: String::new(),
+            // TODO(ilyagr): It would be good if there was a way to explicitly specify that a
+            // certain tool (e.g. vscode as of this writing) cannot be used as a diff editor (or a
+            // diff tool). One possible TOML syntax would be `edit-args = false`.
```

Comment:
Perhaps, `.edit-args = []` can be used to explicitly turn off diff editor.

Reply:
I'm not sure about this, but I added your suggestion to the comment.

I think `false` is clearer to the user, but `[]` might be less awkward to implement. Either will work in the short term, in the long term perhaps we should move on to something like `edit = { args = [...] }` vs `edit = {disabled = true}`.